.github/workflows/build.yml

```diff
@@ -34,6 +34,7 @@ jobs:
     env:
       TEST_AUTH_TOKEN: ${{ secrets.ALPHA_TEST_AUTH_TOKEN }}
       TEST_CACHE_NAME: js-integration-test-ci
+      TEST_SESSION_TOKEN: ${{ secrets.ALPHA_TEST_SESSION_TOKEN }}
```

Comment:
These session tokens expire after a few hours, we would need a way to programatically regenerate them in order to have these integration tests run consistantly

Reply:
if we _really_ wanted to have session token integration tests, we could have selenium log into our console, grab a session token and use that. But we can probably set that up later.

Reply:
for sure, its a really good test to have!

Reply:
Leave the test and skip it for now.

Reply:
we should remove this env var from the github thing for now right?